networkit/graph.pyx

```diff
@@ -665,6 +665,186 @@ cdef class Graph:
 			yield dereference(it)
 			preincrement(it)
 
+	def attachNodeAttribute(self, name, ofType):
+		"""
+		Attaches a node attribute to the graph and returns it
```

Comment:
This and the following comments refer to the documentation. While the build on CI is successful, we currently not check (and thats not that easy) the correct syntax and needed doc-strings for rendering. When I build the branch locally, the following (errornous) output is generated:

![Bildschirmfoto 2022-02-25 um 12 32 45](https://user-images.githubusercontent.com/4864162/155708464-3f468124-5b15-4a30-a2e5-8fd017543251.png)
![Bildschirmfoto 2022-02-25 um 12 33 07](https://user-images.githubusercontent.com/4864162/155708469-52f97e17-be66-4bd7-ab5a-d4f0ba97b7df.png)

Add `attachNodeAttribute(name, ofType)` as the first line of the doc (enables declaration visibility in rendered doc).

Reply:
Thanks for testing the docs, now it should be fine.

Reply:
Sorry for being nitpicky, but still looks a bit odd. (I also could have been clearer with my comments)

<img width="870" alt="Bildschirmfoto 2022-02-25 um 13 31 11" src="https://user-images.githubusercontent.com/4864162/155715672-9aef8701-36f5-42b3-9be7-a41cf41902cd.png">

Besides the wrong look description for `attachNodeAttribute`, we also have missing punctuation marks and the parameter description of `detach` is still wrong.

Reply:
Also in general - would it make sense to rename `detach` to `detachNodeAttribute` to be in line with `attachNodeAttribute`?